index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1 id="title">freeCodeCamp Survey Form </h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form id="survey-form">
+      <label id="name-label">Name</label>
+      <input id="name" type="text" placeholder="Enter Name" required/>
+      <label id="email-label">Email</label>
+      <input id="email" type="email" placeholder="Enter Email" value="" required/>
+      <label id="number-label">Number</label>
+      <input id="number" type="number" min=10 max=20 type="number" placeholder="no" value=""/>
+      <select id="dropdown" placeholder="gg">
+          <option value="op1">Option1</option>
+         <option value="op2">Option2</option>
+      </select>
+      <input type="radio" value="" name=""/>
+      <input type="radio" value="" name=""/>
+      <input type="checkbox" value="" name="">frontend</input>
+      <input type="checkbox" value="" name="">backend</input>
+      <textarea name="comment">Enter text here...</textarea>
+      <input id="submit" type="submit"/>
+    </form>
+  </body>

```

Comment:
- [ ] **Good use of GitHub inline comments 👍** However, It would be better to refactor your review with a suggestion instead of giving an order 😎😎
For example:

This is a polite suggestion: "I think it would be better to use the appropriate tags for each elements ...."

This reads as an order: "Use the appropriate tags for each.."

Reply:
Thanks for the Review, I will do that.